neo4j/_async/driver.py

```diff
@@ -206,6 +215,61 @@ def driver(cls, uri, *, auth=None, **config) -> AsyncDriver:
             return cls.neo4j_driver(parsed.netloc, auth=auth,
                                     routing_context=routing_context, **config)
 
+    @classmethod
+    def bookmark_manager(
+        cls,
+        initial_bookmarks: t.Mapping[str, t.Union[Bookmarks,
+                                                  t.Iterable[str]]] = None,
+        bookmarks_supplier: _T_BmSupplier = None,
+        bookmarks_consumer: _T_BmConsumer = None
+    ) -> AsyncBookmarkManager:
+        """Create a default :class:`.AsyncBookmarkManager`.
+
+        Basic usage example to configure the driver with the default
+        bookmark manger implementation so that all work is automatically
+        causally chained (i.e., all reads can observe all previous writes
+        even in a clustered setup)::
+
+            import neo4j
+
+            driver = neo4j.AsyncGraphDatabase.driver(
+                uri, auth=..., # ...
+                bookmark_manager=neo4j.AsyncGraphDatabase.bookmark_manager(
+                    # ... configure the bookmark manager
+                )
+            )
+
+        :param initial_bookmarks:
+            The initial set of bookmarks. The default bookmark manager will
```

Comment:
```suggestion
            The initial set of bookmarks. The built-in implementation of the bookmark manager will
```
just saying "The default bookmark manager" may leave people to think it's enabled by default.
The default bookmark manager is currently a no-op.